charts/flyte-binary/templates/configmap.yaml

```diff
@@ -190,6 +190,8 @@ data:
       security:
         useAuth: true
   {{- end }}
+  005-flyte-binary-config.yaml: |
```

Comment:
Would it make sense for this to be merged into `000-core.yaml` instead?

Reply:
moved up to `000-core.yaml`; I agree that is a better place for it to live.